test.sh

```diff
@@ -78,6 +91,26 @@ function run_and_expect_silence() {
   rm "${result_file}"
 }
 
+configure_database_endpoints() {
+  target_dir="mariadb"
+  dbconfig_target="dbconfig.mariadb.yml"
+  export MYSQL_ADDR="boulder-proxysql:6033"
```

Comment:
hyper-nit: DB_ADDR, since neither MariaDB nor ProxySQL nor Vitess is truly "MYSQL".